app/models/concerns/hud_reports/clients.rb

```diff
@@ -26,12 +26,23 @@ module HudReports::Clients
       {
         'Yes' => column.eq(1),
         'No' => column.eq(0),
-        'Client Doesn\'t Know/Client Refused' => column.in([8, 9]),
+        label_for(:dkptr) => column.in([8, 9]),
         'Data Not Collected' => column.eq(99).or(column.eq(nil)),
         'Total' => Arel.sql('1=1'),
       }
     end
 
+    # override HudReports::Clients concern
+    def yes_know_dkn_clauses(column)
+      {
+        'Yes' => column.eq(1),
+        'No' => column.eq(0),
+        label_for(:dkptr) => column.in([8, 9]),
+        label_for(:data_not_collected) => column.eq(99).or(column.eq(nil)),
+        'Total' => Arel.sql('1=1'),
+      }
+    end
```

Comment:
Pretty sure we can just put this in `HudReports::Clients`  I'm not seeing a functional difference.